README.md

```diff
@@ -1,14 +1,48 @@
-# voyage-tasks
+![GitHub Contributors](https://img.shields.io/github/contributors/chingu-voyages/v36-toucans-team-04?style=plastic)
+![GitHub Language Count](https://img.shields.io/github/languages/count/chingu-voyages/v36-toucans-team-04?style=plastic)
+![GitHub last commit](https://img.shields.io/github/last-commit/chingu-voyages/v36-toucans-team-04?style=plastic)
 
-Your project's `readme` is as important to success as your code. For 
-this reason you should put as much care into its creation and maintenance
-as you would any other component of the application.
+# Falling Words Game
 
-If you are unsure of what should go into the `readme` let this article,
-written by an experienced Chingu, be your starting point - 
-[Keys to a well written README](https://tinyurl.com/yk3wubft).
+A fun and interactive way to improve your typing skill!
 
-And before we go there's "one more thing"! Once you decide what to include
-in your `readme` feel free to replace the text we've provided here.
+## Overview
 
-> Own it & Make it your Own!
+This Javascript game was created for the Chingu Voyage 36. Our team wants to provide a game in which players can improve their typing skill. We are designing this game to cater to players at different skill levels. Whether you are a beginner typist or an experienced typist, we will make sure that you are challenged! The main goal of this game is to type the words that are falling from the top before they reach the bottom. The difficulty level will increase as you progress through the game. At the end of the game, you can review your performance through comprehensive feedback.
+
+>Live Link: [https://chingu-voyages.github.io/v36-toucans-team-04/](https://chingu-voyages.github.io/v36-toucans-team-04/)
+
+## Features
+| Key Feature                                              | Status           |
+| -------------------------------------------------------- | ---------------- |
+| Landing Screen                                           | Work In Progress |
+| Tutorial Screen                                          | Work In Progress |
+| Falling words that player must type                      | Work In Progress |
+| A set number of chances that players can miss words      | Work In Progress |
+| Spawning of bonus word that can reward additional chance | Work In Progress |
+| Increasing difficulty level                              | Work In Progress |
+| Capability to select starting difficulty level           | Work In Progress |
+| GameOver screen with player's performance details        | Work In Progress |
+| Fun Facts about Typing                                   | Work In Progress |
+| Credits Screen                                           | Work In Progress |
+
```

Comment:
You should add a "Running the project" section as specified in here: https://medium.com/chingu/keys-to-a-well-written-readme-55c53d34fe6d (Jim sent me this link for creating the README.md for my SoloProject)"

Reply:
Got it. I will add that section.